### ISSUE TITLE
Have a reverse mode for creamsody mode line

### ISSUE THREAD

**Issue by dakrone** [emacsfodder/emacs-theme-creamsody#3]
To me when using a dark theme, I'd like to have the active buffer's modeline be the brightest. It'd be great if there were a `(creamsody-reverse-modeline)` that did what the regular modeline does, but makes the inactive buffers dark and the active buffer the teal-ish color.

What do you think?

**Comment by jasonm23** [comment]
Fixed in a0071bf037a7f2d87097918fe5338e23f4736edd

**Comment by jasonm23** [comment]
@dakrone we now have a selection of four modeline themes named 

```
(creamsody-modeline-one)
(creamsody-modeline-two)
(creamsody-modeline-three)
(creamsody-modeline-four)
```

You can check them out on the README view here on github.
